apps/gitness/src/pages-v2/pull-request/stores/pull-request-store.tsx

```diff
@@ -0,0 +1,67 @@
+import { create } from 'zustand'
+
+import { ListPullReqOkResponse } from '@harnessio/code-service-client'
+
+import { timeAgoFromEpochTime } from '../../../pages/pipeline-edit/utils/time-utils'
+import { PageResponseHeader } from '../../../types'
+
+interface PullRequestType {
+  is_draft?: boolean
+  merged?: number | null // TODO: Should merged really be all these??
+  name: string | undefined
+  number?: number
+  sha?: string
+  author?: string
+  reviewRequired: boolean
+  tasks?: number
+  source_branch?: string
+  timestamp: string
+  comments?: number
+  state?: string
+  labels?: {
+    text: string
+    color: string
+  }[]
+}
+interface PullRequestStore {
+  pullRequests: PullRequestType[] | null
```

Comment:
i did this because of the data type of pullrequest , it returns a null from the api @knagurski , will be removed once the api is fixed
https://harness.atlassian.net/browse/PIPE-23359?focusedCommentId=906916